src/trackers/TVC.py

```
@@ -255,105 +326,183 @@ async def upload(self, meta, disctype):
+                    with open(torrent_path, "rb") as open_torrent:
+                        files = {'torrent': open_torrent}
+                        response = await client.post(
+                            self.upload_url,
+                            files=files,
+                            data=data,
+                            headers={'User-Agent': 'Mozilla/5.0'},
+                            params={'api_token': self.config['TRACKERS'][self.tracker]['api_key'].strip()}
+                        )
+
+                if response.status_code != 200:
+                    raise RuntimeError(f"TVC upload failed with HTTP {response.status_code}")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)